composer.json

```diff
@@ -12,10 +12,10 @@
     ],
 
     "require": {
-        "php": ">=7.1.3",
-        "illuminate/support": "~5.7.0|~5.8.0",
-        "illuminate/database": "~5.7.0|~5.8.0",
-        "illuminate/events": "~5.7.0|~5.8.0"
+        "php": ">=7.2",
+        "illuminate/support": "~5.7.0|~5.8.0|~6.0.0",
```

Comment:
`~6.0.0` should be replaced with `^6.0`

Reply:
@mewebstudio could you please change that?

Reply:
Fixed.